schema/v5.0/CVE_JSON_5.0.schema

```diff
@@ -421,14 +417,13 @@
                 },
                 "updated": {
                     "$ref": "#/definitions/timestamp",
-                    "description": "Timestamp to be set by system of record at time of submission. If  UPDATED is provided to the system of record it will be replaced by the timestamp at the time of submission. If a provider has multiple contributions, they shall be consolidated to a final single contribution before submission, or the system of record will reject the input with, Rejected – simultaneous contributions by a single provider."
+                    "description": "Timestamp to be set by the system of record at time of submission. If updated is provided to the system of record it will be replaced by the current timestamp at the time of submission. If a provider has multiple contributions, they shall be consolidated to a final single contribution before submission, or the system of record will reject the input with, Rejected \u2013 simultaneous contributions by a single provider."
                 }
             },
-            "required": [
-                "id"
-            ]
+            "required": ["id"]
         },
         "cnaContainer": {
+            "$comment": "TODO: description needed",
```

Comment:
An object containing the vulnerability information provided by a CVE Numbering Authority (CNA).  There can only be one cnaContainer per CVE record since there can only be one assigning CNA.  The cnaContainer must include the required information defined in the CVE Rules, which includes a product, version, problem type, prose description, and a reference.